src/LanguageServer/ProtocolUnitTests/Configuration/DidChangeConfigurationNotificationHandlerTest.cs

```diff
@@ -107,13 +107,13 @@ public class A { }";
         public void VerifyLspClientOptionNames()
         {
             var actualNames = DidChangeConfigurationNotificationHandler.SupportedOptions.Select(
-                DidChangeConfigurationNotificationHandler.GenerateFullNameForOption).OrderBy(name => name).ToArray();
-            // These options are persist in the LSP client. Please make sure also modify the LSP client code if these strings are changed.
+                DidChangeConfigurationNotificationHandler.GenerateFullNameForOption);
+            // These options are persisted by the LSP client. Please make sure also modify the LSP client code if these strings are changed.
             var expectedNames = new[]
             {
                 "symbol_search.dotnet_search_reference_assemblies",
-                "implement_type.dotnet_insertion_behavior",
-                "implement_type.dotnet_property_generation_behavior",
+                "code_actions.type_members.dotnet_member_insertion_location",
+                "code_actions.type_members.dotnet_property_generation_behavior",
```

Comment:
why did these change?

Reply:
The name is generated from the option group names and the editorconfig name. 

In editorconfig we now have:

```
#### .NET Code Actions ####

# Type members
dotnet_hide_advanced_members = true|false
dotnet_member_insertion_location = with_other_members_of_the_same_kind | at_the_end
dotnet_property_generation_behavior =  prefer_throwing_properties | prefer_auto_properties
```

I thought it would make sense to have the VS Code setting reflect the same grouping, but we can keep `implement_type` group as well.

Reply:
I'll keep `implement_type` group. Looking at the other options it seems we don't group them the same way between editorconfig and VS Code settings anyways.

Reply:
I'm gonna remove the `code_actions.` prefix as other options don't have 3 levels of nesting.

Reply:
The change from `implement_type` prefix to `type_members` is to generalize the option group to include `dotnet_hide_advanced_members`. 
Also, `dotnet_member_insertion_location` and `dotnet_property_generation_behavior` don't look like options specific to `implement_type` feature. They could be used by other features that add members/properties to types.

Reply:
wfm.  could you update the OP to reflect this.